thoth/storages/graph/postgres.py

```diff
@@ -50,6 +50,7 @@
 from sqlalchemy.orm import sessionmaker
 from sqlalchemy.orm.session import Session
 from sqlalchemy.dialects.postgresql import insert
+from thoth import python
```

Comment:
Is this import needed?

Reply:
mmm nope, not sure why is there actually :D Thanks for spotting it :+1: